core/tests/protractor_utils/admin.js

```diff
@@ -59,6 +59,21 @@ var editConfigProperty = function(
   );
 };
 
+var updateRole = function(name, newRole) {
+  general.waitForSystem();
+  browser.get(general.ADMIN_URL_SUFFIX);
+  element(by.css('.protractor-test-admin-roles-tab')).click();
+  browser.waitForAngular();
+  //change values for update form and submit it.
+  element(by.css('.protractor-update-form-name')).sendKeys(name);
+  var select = element(by.model('updateFormValues.newRole'));
+  var searchKey = '[label="' + newRole + '"]'
+  select.$(searchKey).click();
```

Comment:
This doesn't look like a "human action" thing to me. How do we handle other dropdowns in protractor? Can't we do something like "get the option matching the given human-readable label"?